### DISCUSSION TITLE
Urodzinowy meetup z okazji 5 lecia naszej grupy!

### DISCUSSION THREAD

**Discussion by mgrabarz** [AzureCommunityPL/group#156] [Wydarzenia]
Cześć ekipo!

W drugą środę grudnia będziemy świętować okrągłą rocznicę naszej grupy - 9.12 mija dokładnie 5 lat od [pierwszego wydarzenia](https://www.meetup.com/Microsoft-Azure-Users-Group-Poland/events/226809106/)!

Ten meetup musi być mocno nieszablonowy. Czy zrobimy merytoryczną dyskusję, czy wspólnie online wypijemy urodzinowego szampana, a może zwyczajnie wszyscy podyskutujemy na Teams?! Opcji jest wiele, a czasu na organizację mało. 

**Podzielcie się proszę swoimi pomysłami i opiniami.** Może ktoś chciałby też dołożyć organizacyjną cegiełkę? Wraz z resztą organizatorów czekam na Wasze opinie!

**Comment by mgrabarz** [comment]
Pewnie spytacie dlaczego 2gie wydarzenie uważamy za pierwsze - tak to już od dawna przyjęliśmy że pierwsze, październikowe uznajemy jako "watching party" innej imprezki i nie doliczamy :)

**Comment by aryll** [comment]
@mgrabarz a może jakąś retrospekcję? już widzę jak opowiadacie "jak byliśmy piękni i młodzi to.."  😂

**Reply by eklime** [reply]
Ahaha...jaaasnee. Tylko kto miałby o tym opowiadać? Kogo wybrać, jakim kluczem, jakie pytania? Tak wiele pytań, tak mało odpowiedzi...

**Reply by maciejgos** [reply]
Można po stażu w Azure 😉😀. 

@mgrabarz ja osobiście byłbym po prostu za "pogadaniem" na Teams. Można by tak luźno pogadać co się przez te 5 lat zmieniło w Azure oraz jaka jest przyszłość chmury.

**Reply by eklime** [reply]
@maciejgos  przyszłość chmury? To o tym ktoś jeszcze chce słuchać? Ostatnie 5 lat to też obecnie nie najbezpieczniejszy okres retrospektywy 🤣 No ale nic to, zbieramy pomysły dalej.

**Reply by kaluzaaa** [reply]
Byle @mgrabarz nie wspominał naszego pierwszego biznesowego spotkania w kontekście Azure😂

**Reply by mgrabarz** [reply]
W zasadzie to ja bym może i chciał zrobić Teamsa, ale boje się że każdy każdego będzie przekrzykiwał...

Pomysł na jakieś Q&A z przeszłości grupy, wspomnienia itp w sumie niezły. :)

**Reply by aryll** [reply]
Zawsze można zrobić jeszcze "konkursy" np. 
- ilu było uczestników pierwszego meetupu 
- ilu prezenterów / prezentacji było przez 5 lat
- lub kto zrobił najwięcej prezentacji o tym samym tylko pod różnym tematem 😁 

jako taka śmieszna zgadywanka - nic poważnego

**Reply by mgrabarz** [reply]
Ja tam nie znam prawdziwych odpowiedzi 🤷

**Comment by kamil-mrzyglod** [comment]
Zawsze można zrobić panel z predykcjami czy spojrzeniem na kolejne 5 lat. Niczym sztandarowa rozmowa kwalifikacyjna: "Gdzie Pan/Pani widzi siebie za 5 lat" 😉

**Comment by eklime** [comment]
No i co ustaliliśmy?

**Reply by aryll** [reply]
dobre pytanie, zwłaszcza że na meetup'ie nie ma jeszcze żadnego wydarzenia a to już za tydzień 😉

**Reply by maciejgos** [reply]
póki co pomysłów brak oprócz tego co już tutaj napisaliśmy 😔

**Comment by mgrabarz** [comment]
Jednym z pomysłów jest sesja Q&A na dowolne Azurowe tematy. Zasady gry są w sumie proste:
- Zadajemy pytania związane z Azure
- Możecie wskazać dowolną osobę (spośród całego community) jako odpowiadającego.
- Jeżeli tej osoby nie będzie na spotkaniu, organizatorzy postarają się na nie sami odpowiedzieć.
- Można zgłosić dowolną ilość pytań tutaj: https://azurecommunitypl.typeform.com/to/BZ7cuI9S

**Reply by aryll** [reply]
ale robimy to w formie "spotknia (np. na teams) czy w formie streamu?

**Reply by eklime** [reply]
> ale robimy to w formie "spotknia (np. na teams) czy w formie streamu?

Będzie Teams ;)

**Reply by RogalaPiotr** [reply]
> ale robimy to w formie "spotknia (np. na teams) czy w formie streamu?

jak Marek napisał wyżej, ku mojemu zdziwieniu jest tam słowo Teams ;)

**Comment by RogalaPiotr** [comment]
Ja bym zaproponował żebyśmy zrobili coś na zasadzie listy/tematów z osiągnieciami w ostatnim roku. Czego się nauczyłem i co udało mi się zbudować w Azure. Myślę że to będzie fajna burza mózgów.

**Comment by mgajownik** [comment]
Od dłuższego czasu chodzi mi po głowie taka mała gala na koniec roku, czyli uhonorowania ciekawych wdrożeń, ich wykonawców w formie głosowania społeczności w danej kategorii np. migracja do Azure, najfajniejsze wydażenie itp.
Tak wiem czasu na to nie ma obecnie.
I potrzeba by była zgoda firm na publikowanie projektów jakie robimy.

**Reply by RogalaPiotr** [reply]
To prywatny call... po co zgody? :P @mgrabarz

**Reply by mgrabarz** [reply]
No to zaczynasz od Nordcloud ;)

**Reply by RogalaPiotr** [reply]
> No to zaczynasz od Nordcloud ;)

chyba Justcloud 😄

**Reply by aryll** [reply]
W całej idei najbardziej podoba mi się słowo 'gala' - tak jakby miał być raz w roku kiedy każdy ' porządnie' się ubierze 😄

**Reply by maciejgos** [reply]
Gala mistrzów 😀

**Reply by eklime** [reply]
Gala to dość niebezpieczne słowo...jeszcze autokorekta wskoczy i będzie przypał 🤭

**Reply by kaluzaaa** [reply]
@aryll znam tu osoby częściej niż raz do roku się ładnie ubierają... ale kogoś to czasem drogo kosztuje 😂

**Reply by aryll** [reply]
@kaluzaaa no w sumie strach się spytać ale: jakieś szczegóły? Bo nawet nie wiem do kogo się teraz uśmiechamy 😉

**Reply by kaluzaaa** [reply]
@aryll jaki strach :D Mówię o np. Garnitur i krawat 🙈

**Reply by maciejgos** [reply]
@kaluzaaa a czyli, że się wszyscy ładnie ubieramy na meetup 🤣

**Comment by eklime** [comment]
:(
![image](https://user-images.githubusercontent.com/12485979/101149802-b6947200-361f-11eb-91d2-9af3fff40d6d.png)

https://azurecommunitypl.typeform.com/to/BZ7cuI9S
